### PR TITLE
Made few changes for views/components that did not conform to the cor…

### PR DESCRIPTION
**Pull request by ChristianTherkelsen**
…rect ColorProvider colors

# Why?

Enable branding for NMP

# What?

Found couple places where the correct colors was not applied or used. 

# Version Change

This is a patch release

# UI Changes
FINN
No changes to FINN as the updated colors provide same colors as before 

Tori
These four places are modified for Tori branding (Ignore the blue checkmarks as those branded via the ImageProvider). I applied Tori colors to the Demo app to identify these components

The following are now brandable:
- CheckmarkTitleView
- Confettis 
- Settings icon 
- SelectionListView

Screenshot 2023-10-03 at 14 26 43 Screenshot 2023-10-03 at 14 27 04 Screenshot 2023-10-03 at 14 26 24 Screenshot 2023-10-03 at 14 27 23

### CODE REVIEW

File: FinniversKit/Sources/Components/CheckmarkTitleView/CheckmarkTitleView.swift [hunk 1]
```diff
@@ -4,9 +4,10 @@
 
 public class CheckmarkTitleView: UIView {
     private lazy var checkmarkImageView: UIImageView = {
-        let image = UIImage(named: .check)
+        let image = UIImage(named: .check).withRenderingMode(.alwaysTemplate)
```
Comment (ChristianTherkelsen) [inline FinniversKit/Sources/Components/CheckmarkTitleView/CheckmarkTitleView.swift:7]: This checkmark was always FINN blue

File: FinniversKit/Sources/Components/SelectionListView/Subviews/SelectionListItemView.swift [hunk 2]
```diff
@@ -197,7 +197,7 @@ class SelectionListItemView: UIView {
             if self.configuration.isIndependent {
                 self.contentView.layer.borderColor = self.isSelected ? .btnAction : .borderDefault
             } else {
-                self.backgroundColor = self.isSelected ? .btnPrimary : .clear
+                self.backgroundColor = self.isSelected ? .nmpBrandDecoration : .clear
```
Comment (ChristianTherkelsen) [inline FinniversKit/Sources/Components/SelectionListView/Subviews/SelectionListItemView.swift:200]: blue600 for FINN and Petrolium for Tori

File: FinniversKit/Sources/Fullscreen/ConfettiView/ConfettiView.swift [hunk 3]
```diff
@@ -12,8 +12,8 @@ public class ConfettiView: UIView {
     ]
 
     private let confettiColors: [UIColor] = [
-        .blue600,
-        .aqua400,
+        .nmpBrandDecoration,
```
Comment (ChristianTherkelsen) [inline FinniversKit/Sources/Fullscreen/ConfettiView/ConfettiView.swift:15]: This ensures we have branded colors for FINN and Tori for the confetti view

File: FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift [hunk 4]
```diff
@@ -81,6 +81,10 @@ extension CGColor {
     public class var textTertiary: CGColor { UIColor.textTertiary.cgColor }
     public class var textToast: CGColor { UIColor.textToast.cgColor }
     public class var marketplaceNavigationBarIcon: CGColor { Config.colorProvider.marketplaceNavigationBarIcon.cgColor }
+    public class var nmpBrandColorPrimary: CGColor { Config.colorProvider.nmpBrandColorPrimary.cgColor }
```
Comment (adriansergheev) [inline FinniversKit/Sources/DNA/Color/UIColor+FinniversKit.swift:84]: should this be into FinniversKit?

**Comment by ChristianTherkelsen** [comment]
Looking at failing tests 👀

**Review by adriansergheev — APPROVED** [review]

**Pull request merged**